package-lock.json

```diff
@@ -1,8 +1,9143 @@
 {
   "name": "pull-request-community",
   "version": "0.1.0",
-  "lockfileVersion": 1,
+  "lockfileVersion": 2,
   "requires": true,
+  "packages": {
```

Comment:
@lurx 
I prefer to not download all MUI for only for buttons, can you add that buttons as an SVG?

Reply:
MDI was already used, in the github link and the cards icons